### PR TITLE
Minor changes for clarity

### PR DESCRIPTION
**Pull request by jboxman**
@openshift/team-documentation, PTAL.

### CODE REVIEW

File: getting_started/install_openshift.adoc [hunk 1]
```diff
@@ -159,7 +159,7 @@ $ for host in master.openshift.example.com \
 
 . See
 xref:../install/example_inventories.adoc#install-config-example-inventories[Example
-Inventory Files] and select an example that closest matches your desired cluster
+Inventory Files] and select an example that most closely matches your desired cluster
```
Comment (kalexand-rh) [inline getting_started/install_openshift.adoc]: s/an/the

File: getting_started/install_openshift.adoc [hunk 2]
```diff
@@ -173,8 +173,8 @@ xref:../install/configuring_inventory_file.adoc#install-config-configuring-inven
 for full documentation on available inventory variables.
 ====
 
-. Edit the example inventory to use your host names, then save it to a file
-(default location is *_/etc/ansible/hosts_*).
+. Edit the example inventory to use your host names, and then save it to a file
+(the default location is *_/etc/ansible/hosts_*).
```
Comment (kalexand-rh) [inline getting_started/install_openshift.adoc]: Rewrite this to not use parentheses, per IBM Style.

File: getting_started/install_openshift.adoc [hunk 3]
```diff
@@ -199,7 +199,7 @@ basic authentication, user access, and routes.
 {product-title} provides two command line utilities to interact with it.
 
 * `oc`: for normal project and application management
-* `oc adm` or `oadm`: for administrative tasks
+* `oadm` or `oc adm`: for administrative tasks
```
Comment (kalexand-rh) [inline getting_started/install_openshift.adoc]: `oadm` was deprecated in 3.9. You should remove it from 3.9 on.

File: getting_started/install_openshift.adoc [hunk 4]
```diff
@@ -227,7 +227,7 @@ option to either create new roles or define a policy that allows anyone to log
 in (to start you off).
 
 Before you do anything else, log in at least one time with the default
-*system:admin* user, on the master run the following command.
+*system:admin* user. On the master run the following command:
```
Comment (kalexand-rh) [inline getting_started/install_openshift.adoc]: s/master/master,

**Review by kalexand-rh — COMMENTED** [review]
I have more changes. Please update the labels and milestones.

File: getting_started/install_openshift.adoc (hunk 3 above)
Comment (jboxman) [reply getting_started/install_openshift.adoc]: If I'm reading `diff` between branches correctly, this looks like it is correct in 3.11, but not in master. I'll dig around some more and resolve cherry-pick conflicts as needed.

**Comment by jboxman** [comment]
@kalexand-rh, updated. PTAL. Thanks.

**Comment by kalexand-rh** [comment]
@jboxman, LGTM. Thanks!

**Pull request merged**